lib/test/KettleTestUtils.js

```diff
@@ -265,22 +265,21 @@ kettle.test.testDefToServerEnvironment = function (testDef) {
  * module's test suites run in aggregate. The test definitions will be transformed and then contributed
  * to the current queue of asynchronously resolving tests.
  *
- * @param testDefs {Object} or {Array of Object} an array of objects, each representing a test fixture
- * @param transformer {Function} or {Array of Function} an array of transform functions, accepting an object representing a test fixture and returning a "more processed" one. The entire chain
- * of functions will be applied to each member of <code>testDefs</code>, with the result that it becomes a fully fleshed out TestCaseHolder as required by Fluid's
+ * @param {Object|Object[]}testDefs - an array of objects, each representing a test fixture
+ * @param {Function|Function[]} transformer - an array of transform functions, accepting an object representing a test fixture and returning a "more processed" one. The entire chain
+ * of functions will be applied to each member of <code>testDefs</code> with the result that it becomes a fully fleshed out TestCaseHolder as required by Fluid's
  * <a href="http://wiki.fluidproject.org/display/docs/The+IoC+Testing+Framework">IoC Testing Framework</a>
  */
-
 kettle.test.bootstrap = function (testDefs, transformer) {
     var transformArgs = [fluid.makeArray(testDefs)].concat(fluid.makeArray(transformer));
     var tests = fluid.transform.apply(null, transformArgs);
-    return fluid.test.runTests(tests);
+    fluid.test.runTests(tests);
 };
 
 /** As for kettle.test.bootstrap, only transform the supplied definitions by converting them into kettle
```

Comment:
This seems a little tricksy.  This only passes because of the intervening space.  IMO in that this is not a Javadoc but does describe the function, it should be devolved into a normal block comment without the leading double-asterisk.